### ISSUE TITLE
Videos coming soon? :-)

### ISSUE THREAD

**Issue by erichodges** [ndimatteo/HULL#8]
I'm really looking forward to the videos on this project! I hope you can get them done soon :-).

**Comment by ndimatteo** [comment]
Thanks @erichodges! Updates about the video releases will be coming soon, hang tight! 🤘

**Comment by bobinska-dev** [comment]
Same here! This will be wonderful to watch

**Comment by ovsw** [comment]
Any plans on adding the videos still? Thanks for the great work!

**Comment by ndimatteo** [comment]
Hey there @ovsw apologies I haven't been able to provide the video guides yet!

Shopify made major changes to their UI while I was creating them last year and I haven't had an opportunity to re-record them and edit them just yet.

I still have plans to release these, but would like to prioritize updating the repo with new features and general housekeeping first. I'd say after the new year to expect the video guides to come together!

Appreciate the interest and patience to everyone here who's been waiting for them 🙏

**Comment by enterpriseSoftware** [comment]
A Video would go a long way in promoting the use of the template.  The tutorial is a little hard to follow if you have never done it.

**Comment by ndimatteo** [comment]
@enterpriseSoftware videos are planned for the next major release, as a lot has changed over the last 1-2 years with Shopify, Sanity, and Next.js. Stay tuned!

**Comment by louishinohara** [comment]
looking forward to seeing your changes. I tried to run the application locally but encountering some issue due to deprecated shopify-buy API's.
